.github/workflows/ci.yml

```diff
@@ -106,6 +107,8 @@ jobs:
           flavor: |
             latest=${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}
           images: |
+            ghcr.io/qdm12/gluetun
+            ghcr.io/qdm12/private-internet-access
```

Comment:
Let's keep it to gluetun; private-internet-access is for retro-compatibility only

```suggestion
```